aiida/common/progress_reporter.py

```diff
@@ -174,3 +174,26 @@ def set_progress_bar_tqdm(bar_format: Optional[str] = TQDM_BAR_FORMAT, leave: Op
     """
     from tqdm import tqdm
     set_progress_reporter(tqdm, bar_format=bar_format, leave=leave, **kwargs)
+
+
+def create_callback(progress_reporter: ProgressReporterAbstract) -> Callable[[str, Any], None]:
```

Comment:
or make it a method of the progress reporter?

Reply:
tqdm does not have a `create_callback` method though. The only way this could work is if you dynamically added the method in `set_progress_bar_tqdm`, which is a bit hacky